app/components/ImageAssets.tsx

```diff
@@ -49,6 +49,62 @@ export const NoblocksLogo = (props: ImageProps) => {
   );
 };
 
+export const NoblocksBetaLogo = (props: ImageProps) => {
```

Comment:
why `NoblocksBetaLogo`

should be NoblocksLogo yea?

Reply:
there's the normal noblocks logo, this one is the one with the `Beta` tag